client/my-sites/site-settings/podcasting-details/not-supported.jsx

```diff
@@ -0,0 +1,25 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+
+function PodcastingNotSupportedMessage( { translate } ) {
+	return (
+		<div className="podcasting-details__not-supported">
+			<EmptyContent
+				illustration={ '/calypso/images/illustrations/illustration-nosites.svg' }
+				title={ translate( 'Management of podcast settings are not supported on this site.' ) }
```

Comment:
Grammar nitpick:  Management [of podcasting settings] **is** not supported on this site.